pages/common/bgpgrep.md

```diff
@@ -0,0 +1,29 @@
+# bgpgrep
+
+> Filter and print BGP data within MRT dumps.
+> Can read files compressed with gzip, bzip2 and xz.
+> More information: <https://gitea.it/1414codeforge/ubgpsuite>.
+
+- Output all routes:
+
+`bgpgrep {{master6.mrt}}`
+
+- Output all routes received from a specific peer, determined by the AS number:
+
+`bgpgrep {{master4.mrt}} -peer {{64498}}`
+
+- Output all routes received from a specific peer, determined by the peer's IP address:
```

Comment:
Is "determined" appropriate here? What is meant is the following: In a MRT file are routes, which were received from different peers. This command shows all routes received from the peer with IP address `2001:db8:dead:cafe:acd::19e`.

Reply:
```suggestion
- Output all routes received from a specific peer (determined by the peer's IP address):
```

Reply:
I think that brackets would be inappropriate here, because the text that would be in the brackets is not only a hint/note, but essential to understand the command.